recipe_scrapers/_utils.py

```diff
@@ -230,6 +230,19 @@ def normalize_string(string):
     )
 
 
+def normalize_keywords(keywords_str):
+    keywords_list = keywords_str.split(",")
+    seen = set()
+    result = []
+    for keyword in keywords_list:
+        stripped_keyword = keyword.strip()
+        if stripped_keyword.lower() not in seen:
+            seen.add(stripped_keyword.lower())
+            result.append(stripped_keyword)
+    normalized_keywords_str = ",".join(result)
+    return normalized_keywords_str
```

Comment:
Could simplify @jayaddison's suggestion to 

```py
tags = {tag.strip().lower() for tag in raw_tags}
```
and then just convert to a list before returning?

Reply:
Possibly - the order of elements within a `set` isn't guaranteed, though -- and when people express the important properties of their recipes using lists of keywords, I'd expect that in some (maybe not all) cases, some priority can be inferred from the position of each keyword.  I'm not suggesting that we attempt to figure out those priorities, but that we can avoid modifying the source ordering so that if there is any meaning there, it's not lost.